ocaml/utils/profile.ml

```diff
@@ -389,9 +389,15 @@ let column_mapping = [
   `Counters, "counters"
 ]
 
+let chars_to_sanitise_for_csv = [|','; '/'|]
```

Comment:
Out of curiosity, why does `/` need
to be sanitised for CSV?

Reply:
I realise now that the name is slightly misleading. In reality, I want to sanitise a function symbol name such that it can be used in both a CSV field and as a filename (although I don't expect a forward slash in the function symbol name in reality). Since this seems dependent on the implementation of dynamic counters (dumping CSVs for each function with the function symbol name in the filename), I don't need to include this here but rather it can be part of the external implementation.

Reply:
As you prefer, but I was really not
suggesting a change should be
made - just being curious.